marmot.go

```diff
@@ -152,6 +157,10 @@ func main() {
 			}
 
 			os.Exit(0)
+		case <-reloadCh:
+			if err := replicator.ReloadCertificates(); err != nil {
+				log.Info().Msgf("Unable to reload certificates: %s", err)
```

Comment:
`log.Error().Err(err).Msg("Unable to reload certificates")`

Reply:
Oopsie! I'll fix this too